packages/api-server/src/watch.ts

```diff
@@ -1,27 +1,24 @@
-import type { ChildProcess } from 'child_process'
-import { fork } from 'child_process'
-import fs from 'fs'
 import path from 'path'
 
 import chalk from 'chalk'
 import chokidar from 'chokidar'
 import dotenv from 'dotenv'
-import { debounce } from 'lodash'
-import { hideBin } from 'yargs/helpers'
-import yargs from 'yargs/yargs'
 
 import {
   buildApi,
   cleanApiBuild,
   rebuildApi,
 } from '@redwoodjs/internal/dist/build/api'
 import { loadAndValidateSdls } from '@redwoodjs/internal/dist/validateSchema'
-import {
-  ensurePosixPath,
-  getConfig,
-  getPaths,
-  resolveFile,
-} from '@redwoodjs/project-config'
+import { ensurePosixPath, getPaths } from '@redwoodjs/project-config'
+
+import { BuildManager } from './buildManager'
+import { serverManager } from './serverManager'
+
+export type BuildAndRestartOptions = {
+  rebuild?: boolean
+  clean?: boolean
+}
```

Comment:
Do you think we should move this into the buildManager file so that imports are only happening in one direction? I know it's likely not a problem here because its only types.

Reply:
Yeah that's makes more sense.